spec/ios_l10n_helper_spec.rb

```diff
@@ -187,10 +187,31 @@ def file_encoding(path)
   describe '#download_glotpress_export_file' do
     let(:gp_fake_url) { 'https://stub.glotpress.com/rspec-fake-project' }
 
+    it 'sets a predefined User Agent so GlotPress will not rate-limit us' do
+      # Arrange
+      stub = stub_request(:get, "#{gp_fake_url}/fr/default/export-translations/")
+             .with(
+               query: { format: 'strings' },
+               # Note that the syntax below merely checks that the given headers are present in the request,
+               # it does not restrict the request to have only those headers.
+               #
+               # See:
+               # - https://github.com/bblimke/webmock/tree/33d8810c2828fc17010e15cc3f21ad2c726a966f#matching-requests
+               # - https://github.com/bblimke/webmock/issues/276#issuecomment-28625436
+               headers: { 'User-Agent' => 'Automattic App Release Automator; https://github.com/wordpress-mobile/release-toolkit/' }
+             ).to_return(body: 'content')
+      dest = StringIO.new
+      # Act
+      described_class.download_glotpress_export_file(project_url: gp_fake_url, locale: 'fr', filters: nil, destination: dest)
+      # Assert
+      expect(stub).to have_been_made.once
+      expect(dest.string).to eq('content')
+    end
```

Comment:
Nitpick: The android test has this example at the bottom, while here is at the top. _I think we can live with that..._